open_academy/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import open_academy
```

Comment:
Please name the file like the model. 

```suggestion
from . import open_academy_course
```
It is a standard that allows us to know which model is being added or modified, just by looking at the file name.